src/nvim/eval.c

```diff
@@ -2682,19 +2682,21 @@ void ex_call(exarg_T *eap)
     return;
   }
 
-  tofree = trans_function_name(&arg, eap->skip, TFN_INT, &fudi, &partial);
+  tofree = trans_function_name(&arg, false, TFN_INT, &fudi, &partial);
```

Comment:
`assert(!eap->skip)` after the `if (eap->skip)` check to guard against future changes?